model/model-fetchReviews.js

```diff
@@ -6,3 +6,14 @@ exports.fetchReviews = () => {
         return result.rows;
     })
 }
+exports.fetchReviewById = (reviewId) => {
+    if(isNaN(reviewId)){
+        return Promise.reject({
+            msg:"bad request",
+            status:400
+        })
+    }
```

Comment:
Can see you're handling an invalid id here - but you need a test to prove it 👍
Also, you're making work for yourself when the database would tell you if the id was not the correct data type.
Handling it this way would mean you don't have need of a custom error handler... yet